File: tests/go/cases/case_147_dead_branch_global_05.go
package escape_tests

var retainedCase147 = []map[string]string{}

func Case147DeadBranchGlobal05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_branch_global_05",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "dead_branch_global_05:" + raw,
	}
	if false {
		retainedCase147 = append(retainedCase147, payload)
	}
	// SAFE: dead branch retention.
	return payload["checksum"]
}

File: tests/python/cases/case_147_dead_branch_global_05.py
"""
Task case 147: dead_branch_global_05 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_147_dead_branch_global_05(input_data):
    task_name = 'dead_branch_global_05'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if False:
        RETAINED_CACHE['case_147'] = payload
    # SAFE: sink exists only in dead branch and never executes.
    return payload['checksum']
